.github/mergify.yml

```diff
@@ -0,0 +1,10 @@
+pull_request_rules:
```

Comment:
We can't use Mergify for this project (more about this in our team meeting later) but I think I have a different solution that will hopefully make this easy

Reply:
[Updated](https://github.com/cdktf/cdktf-integration-serverless-python-gcp-example/pull/9/commits/f1520ce1724dbaefbe14034975ce6ca7b2a8d888) 👍